rbac/rbac/middleware.py

```diff
@@ -206,13 +207,42 @@ def process_request(self, request):  # pylint: disable=R1710
             user.org_id = json_rh_auth.get("identity", {}).get("org_id") or json_rh_auth.get("identity").get(
                 "internal"
             ).get("org_id")
+
             user_info = json_rh_auth.get("identity", {}).get("user")
             user.username = user_info["username"]
             user.admin = user_info.get("is_org_admin")
             user.internal = user_info.get("is_internal")
             user.user_id = user_info.get("user_id")
             user.system = False
```

Comment:
Maybe we need to check `if user_info:` exists to avoid an exception if not.
I got this doing some tests locally

Reply:
I have changed it so if we don't have both we return a 401. Can you please check that your tests pass now? Thanks!

Reply:
It's working!
Thank you

Reply:
Perfect, thanks for confirming it! :tada: